docker-for-mac/install.md

```diff
@@ -4,82 +4,15 @@ keywords: mac, beta, alpha, install, download
-on [Docker Daemon
-preferences](index.md#daemon-experimental-mode) for experimental
-mode. We recommend that you disable experimental features for apps in
-production.
-
-On both channels, we welcome your <a
-href="troubleshoot/#diagnose-problems-send-feedback-and-create-github-issues">feedback</a>
-to help us as the apps evolve.
-
-For more about Stable and Edge channels, see the
-[FAQs](faqs.md#stable-and-edge-channels).
-
-<table style="width:100%">
-  <tr>
-    <th style="font-size: x-large; font-family: arial">Stable channel</th>
-    <th style="font-size: x-large; font-family: arial">Edge channel</th>
-  </tr>
-  <tr valign="top">
-    <td width="50%">This installer is fully baked and tested. This is the
-    best channel to use if you want a reliable platform to work with. These releases follow the Docker Engine stable releases.<br><br>
-   On this channel, you can select whether to send usage
-   statistics and other data. <br><br>Stable builds are released once per quarter.
-   </td>
-    <td width="50%">This installer provides the latest Edge release of
-    Docker for Mac and Engine, and typically offers new features in development. Use this channel if you want to get experimental features faster, and can weather some instability and bugs. We collect all usage data on Edge releases across the board. <br><br>Edge builds are released once per month.
-    </td>
-  </tr>
-  <tr valign="top">
-  <td width="50%">
-  <a class="button outline-btn" href="https://download.docker.com/mac/stable/Docker.dmg">Get Docker for Mac (Stable)</a>
-  </td>
-  <td width="50%">
-  <a class="button outline-btn" href="https://download.docker.com/mac/edge/Docker.dmg">Get Docker for Mac (Edge)</a>
-  </td>
-  </tr>
-  <tr valign="top">
-  <td width="50%">
-  <a href="https://download.docker.com/mac/stable/Docker.dmg.sha256sum"><font color="#BDBDBD" size="-1">Checksum: Docker.dmg SHA256</font></a>
-  </td>
-  <td width="50%">
-  <a href="https://download.docker.com/mac/edge/Docker.dmg.sha256sum"><font color="#BDBDBD" size="-1">Checksum: Docker.dmg SHA256</font></a>
-  </td>
-  </tr>
-</table>
-
-* Docker for Mac requires OS X El Capitan 10.11 or newer macOS release running on a 2010 or newer Mac, with Intel's  hardware support for MMU virtualization.
-The app runs on 10.10.3 Yosemite, but with limited support. See [What
-to know before you install](#what-to-know-before-you-install) for a full
-explanation and list of prerequisites.
-
-* You can switch between Edge and Stable versions, but you must have only one
-   app installed at a time. Also, you need to save images and export
-   containers you want to keep before uninstalling the current version before
-   installing another. For more about this, see the
-   [FAQs about Stable and Edge channels](faqs.md#stable-and-edge-channels).
+Docker for Mac is the
+[Community Edition (CE)](https://www.docker.com/community-edition)
+of Docker for MacOS. If you already have Docker for Mac installed, skip to
```

Comment:
If they already have it installed, why would they even be on this page? For comparison, I ripped it out here: https://docs.docker.com/docker-for-windows/install/

Reply:
Was been conservative because I thought we had added this info recently. I'll kill that part too 🎉